artworks/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
-
+from art_web_app.models import CustomModel
 # Create your models here.
+from django.contrib.auth.models import User
+from io import BytesIO
+from PIL import Image
+from django.core.files.base import ContentFile
+from utils.constants import THUMB_SIZE
+import os
+
+
+class Categories(CustomModel):
+    category = models.CharField(max_length=50, null=False)
```

Comment:
`name` would be more intuitive. Also note that the default for `null` is `False`.